crates/starknet-server/src/api/json_rpc/endpoints.rs

```diff
@@ -196,10 +196,25 @@ impl JsonRpcHandler {
     /// starknet_getClass
     pub(crate) async fn get_class(
         &self,
-        _block_id: BlockId,
-        _class_hash: ClassHash,
-    ) -> RpcResult<ContractClass> {
-        Err(error::ApiError::ClassHashNotFound)
+        block_id: BlockId,
+        class_hash: ClassHash,
+    ) -> RpcResult<CodegenContractClass> {
+        Ok(self.api.starknet.read().await.get_class(block_id.into(), class_hash)?.try_into()?)
```

Comment:
Why there is no error handling in this method like block not found, class hash not found. This way will use the common StarknetDevnetError

Reply:
Seems weird that we have to do all of that. Why don't we use [StarknetError](https://github.com/xJonathanLEI/starknet-rs/blob/e9bdd7efeae9c76ef62a130ccef3f4355a864797/starknet-core/src/types/codegen.rs#L1026) from `starknet-core`? It has all error & error codes defined & display implemented. Also our errors would be compliant with starknet-rs

Reply:
Done

Reply:
sure, created an issue for it https://github.com/0xSpaceShard/starknet-devnet-rs/issues/130